EventFilter/L1TRawToDigi/plugins/implementations_stage2/BMTFUnpackerOutput.h

```diff
@@ -4,11 +4,17 @@
 #include "BMTFCollections.h"
 
 namespace l1t{
-	namespace stage2{
-		class BMTFUnpackerOutput : public Unpacker
-		{
-			public:
-				bool unpack(const Block& block, UnpackerCollections *coll) override;
-		};
-	}
+  namespace stage2{
+    class BMTFUnpackerOutput : public Unpacker
```

Comment:
Since the code to unpack is the same for both I would keep only one unpacker module and select the unpacking mode with a member variable that you can set in the BMTFSetup.cc.
E.g.:
```
auto outputKalmanMuon = UnpackerFactory::get()->make("stage2::BMTFUnpackerOutput");
outputKalmanMuon->setKalmanFlag(true);
```